__tests__/nc_news_test.test.js

```diff
@@ -89,3 +89,73 @@ describe('GET /api/topics', () => {
             })
         });
     })
+    describe('GET /api/articles', () => {
+        test('Responds with status code 200 and an articles array, with article objects and correct length.' , () => {
+            return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then((response) => {
+                const articles = response.body.articles;
+                expect(Array.isArray(articles)).toBe(true);
+                articles.forEach((article) => {
+                    expect(typeof article).toBe("object")
+                })
+                expect(articles.length).toBe(13);
+            })
+        });
+        test('Returned article objects posses an additional property of \'comment_count\'', () => {
+            return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then((response) => {
+                const articles = response.body.articles;
+                articles.forEach((article) => {
+                    expect(article).toHaveProperty("comment_count");
+                }) 
+            })    
+        })
+        test('Property \'comment_count\', now has a value of the number of comments who share that article ID', () => {
```

Comment:
I would argue this test is optional as your test data could have been a lot bigger.